docs/docs/learn/reference/tools/onboarding.md

```
@@ -5,6 +5,63 @@ sidebar_position: 10
+ - [BOB Mainnet](https://thirdweb.com/bridge?chainId=60808)
+ 
+ ### Universal Bridge Features
+ 
+ - Let users pay for assets in any EVM token on BOB right in your app  
+ - Automatic cross-chain routing for seamless transactions  
+ - Earn from day one with fee-sharing mechanism
+ - Access ready-made UI component for easy implementation  
+ 
+ [Learn more in the full documentation on thirdweb’s Universal Bridge](https://portal.thirdweb.com/connect/pay/overview)
+ 
+ ### Get Started with thirdweb Universal Bridge
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line
null

(MD023, heading-start-left)